src/otx/recipe/keypoint_detection/rtmpose_tiny.yaml

```diff
@@ -0,0 +1,110 @@
+model:
+  class_path: otx.algo.keypoint_detection.rtmpose.RTMPoseTiny
+  init_args:
+    label_info: 17
+
+    optimizer:
+      class_path: torch.optim.AdamW
+      init_args:
+        lr: 0.004
+        weight_decay: 0.0001
+
+    scheduler:
+      class_path: otx.core.schedulers.LinearWarmupSchedulerCallable
+      init_args:
+        num_warmup_steps: 3
+        main_scheduler_callable:
+          class_path: lightning.pytorch.cli.ReduceLROnPlateau
+          init_args:
+            mode: max
+            factor: 0.1
+            patience: 9
+            monitor: val/accuracy
+
+engine:
+  task: KEYPOINT_DETECTION
+  device: auto
+
+callback_monitor: val/accuracy
+
+data:
+  task: KEYPOINT_DETECTION
+  mem_cache_size: 1GB
+  mem_cache_img_max_size: null
+  stack_images: true
+  data_format: coco_person_keypoints
+  unannotated_items_ratio: 0.0
+  image_color_channel: RGB
+  train_subset:
+    subset_name: train
+    batch_size: 32
+    transforms:
+      - class_path: otx.core.data.transform_libs.torchvision.GetBBoxCenterScale
+      # - class_path: otx.core.data.transform_libs.torchvision.RandomFlip
```

Comment:
Could you remove commented code?

Reply:
fixed at 713190f753fa04d865ba9a78ab58328a99517710